src/test/resources/application-integration-test.yml

```diff
@@ -1,13 +1,24 @@
 workspace.integration-test:
   data-repo-instance-names:
+    local: terra
     dev: terra
+    alpha: terra
+    staging: terra
   data-repo-snapshot-id:
+    local: 97b5559a-2f8f-4df3-89ae-5a249173ee0c
```

Comment:
nit: since these same 4 keys are showing up in several objects, it might be better to factor them into a list of objects with {snapshot-id, user-email, wsm-url} and point to a whole object instead of each attribute. Might make it harder to leave one out somewhere.

Reply:
Not all keys are environment dependent though - we could discuss the format of this file outside of this PR so that other people can give their opinions